webapp/packages/product-base/src/manualChunks.ts

```diff
@@ -7,90 +7,60 @@
  */
 import { type PluginOption } from 'vite';
 
-// eslint-disable-next-line arrow-body-style
-export const manualChunks = (): PluginOption => {
-  return [
-    {
-      name: 'manual-chunks',
-      enforce: 'pre',
-      config(config) {
-        return {
-          ...config,
-          build: {
-            ...config.build,
-            rollupOptions: {
-              ...config.build?.rollupOptions,
-              output: {
-                ...config.build?.rollupOptions?.output,
-                manualChunks(id, { getModuleInfo }) {
-                  function isModuleSync(moduleId: string) {
-                    const info = getModuleInfo(moduleId);
-                    if (!info) {
-                      return true;
-                    } // fallback if no info available
-                    if (info.isEntry) {
-                      return true;
-                    }
-                    // info.importers is the list of modules that import this module.
-                    // info.dynamicImporters is the subset of those that import it dynamically.
-                    // If there’s any importer that did a static import, we consider this module sync.
-                    return info.importers && info.importers.some(importer => !info.dynamicImporters.includes(importer));
-                  }
-
-                  const nodeModulesMatch = /[\\/]node_modules[\\/](.*?)[\\/]/.exec(id);
+const vendorChunks = new Map([
+  ['d3-', 'vendor/d3'],
+  ['lodash', 'vendor/lodash'],
+  ['codemirror', 'vendor/codemirror'],
```

Comment:
probably we want to extract react and react-dom as well as some dependencies listed here: `packages/core-cli/configs/excludedFromVendor.js`

(not all of them should be extracted to separate chunks, for example d3 and degree should be in the same chunk because usually used together)

`codemirror` + `@codemirror` + `@lezer`

`leaflet` + `react-leaflet`